io.github.hakuneko.HakuNeko.json

```diff
@@ -0,0 +1,64 @@
+{
+	"app-id": "io.github.hakuneko.HakuNeko",
+	"base": "org.electronjs.Electron2.BaseApp",
+	"base-version": "19.08",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "19.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "hakuneko",
+	"separate-locales": false,
+	"rename-desktop-file": "hakuneko-desktop.desktop",
+	"rename-icon": "hakuneko-desktop",
+	"finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home"
+	],
+	"modules": [
+		{
+			"name": "zypak",
```

Comment:
zypak is part of the electron baseApp now.

Reply:
Ok, I removed the zypak section. The application works without the section.